server/README.md

```diff
@@ -0,0 +1,35 @@
+# Server
+
+This directory contains the server part of audapolis.
+The server provides functionality that was either to annoying to implement in javascript or deemed too costly to perform only in the app.
+
+Currently the main functionality it will provide is transcription:
+With `vosk`, a great python-library for transcribing audio exists.
+However running the speech-to-text on a normal computer takes a lot of time.
+This can be speed up a lot with a GPU.
```

Comment:
does vosk really support that?

Reply:
There is promising work on it in https://github.com/alphacep/vosk-api/issues/74 which looks like it might already be far enough to be used in this project. However until we know that I will reword the readme